### PR TITLE
cases: remove Sharing Data

### PR DESCRIPTION
**Pull request by jorgeorpinel**
per https://github.com/iterative/dvc.org/issues/3186#issuecomment-1036599599

### CODE REVIEW

File: content/docs/use-cases/sharing-data-and-model-files.md [hunk 1]
```diff
@@ -1,99 +0,0 @@
...
-easy to consistently get all your data files and directories into any machine,
-along with matching source code. All you need to do is to setup
-[remote storage](/doc/command-reference/remote) for your <abbr>DVC
-project</abbr>, and push the data there, so others can reach it. Currently DVC
-supports Amazon S3, Microsoft Azure Blob Storage, Google Drive, Google Cloud
-Storage, SSH, HDFS, and other remote locations. The list is constantly growing.
-(For a complete list and configuration instructions, refer to `dvc remote add`.)
-
-![](/img/model-sharing-digram.png) _Data is stored in storage you control,
-separate from code._
```
Comment (jorgeorpinel) [inline content/docs/use-cases/sharing-data-and-model-files.md:13]: I would perhaps consider moving this image to the UC index for now (https://dvc.org/doc/use-cases). WDYT @shcheklein ?

Comment (shcheklein) [reply content/docs/use-cases/sharing-data-and-model-files.md:13]: It seems too specific to me still to be honest

Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-data-and-model-files.md:13]: OK well so far this PR only removes the page (and image) as agreed.

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
- [ ] Add a redirect? Maybe to remote cmd refs.

**Comment by jorgeorpinel** [comment]
>  Redirect to https://dvc.org/doc/command-reference/remote/modify or similar?

See https://github.com/iterative/dvc.org/pull/3327

> Double check basic remote usage is in some intro docs

It's in https://dvc.org/doc/start/data-and-model-versioning#storing-and-sharing

**Pull request merged**